docs/pages/guide/02-start.md

```diff
@@ -0,0 +1,409 @@
+          definition: '{{type}}'
+        }
+        {
+          name: title
+          type: TEXT
+          definition: '{{title}}'
+        }
+        {
+          name: director
+          type: TEXT
+          definition: '{{director}}'
+        }
+        {
+          name: cast
+          type: TEXT
+          definition: '{{cast_list}}'
+        }
+        {
+          name: country
+          type: TEXT
+          definition: '{{country}}'
+        }
+        {
+          name: date_available
+          type: TIME
+          definition: '{{date_added}}'
+          grain: {
+            type: DAY
+          }
+        }
+        {
+          name: release_year
+          type: TIME
+          definition: '{{release_year}}'
+          grain: {
+            type: YEAR
+          }
+        }
+        {
+          name: film_rating
+          type: TEXT
+          definition: '{{rating}}'
+        }
+        {
+          name: genres
+          type: TEXT
+          definition: '{{listed_in}}'
+        }
+        {
+          name: description
+          type: TEXT
+          definition: '{{description}}'
+        }
+      ]
+      measures: [
+        {
+          name: count
+          type: INTEGER
+          definition: 'count({{title_id}})'
+        }
+        {
+          name: total_seasons
+          type: INTEGER
+          definition: "sum(cast (case when {{duration}} like '% Seasons' then REPLACE({{duration}}, ' Seasons', '') else '0' end AS INT))"
+        }
+        {
+          name: movie_duration
+          type: INTEGER
+          definition: "sum(cast (case when {{duration}} like '% min' then REPLACE({{duration}}, ' min', '') else '0' end AS INT))"
+        }
+      ]
+    }
+```
+
+<span class="c11">Yavin pulls and presents all of the semantic model
+metadata from Elide.</span>
+
+<span class="c11"></span>
+
+<img style="border:2px solid black;" src="assets/images/SAI_Model_in_UI.png" width="200" />
+
+<span class="c43">Figure X - Result of the model in the UI</span>
+
+<span class="c25 c36"></span>
+
+<span class="c29">Yavin Quickstart Guide</span>
+-----------------------------------------------
+
+<span class="c11">Here is the complete quickstart guide for setting up
+Yavin with your data source and your data models:</span>
+
+<span class="c11"></span>
+
+1.  <span class="c11">Install Yavin on your local host (Check
+    Installation Guide)</span>
+2.  Upon installation (Step 1) you will have the Yavin repo on your
+    local machine: <span
+    class="c13"><a href="https://www.google.com/url?q=https://github.com/yahoo/navi/&amp;sa=D&amp;ust=1606228594056000&amp;usg=AOvVaw10XIwf--cXX0UU921zZVk3" class="c14">https://github.com/yahoo/navi/</a></span><span
+    class="c11">. Your repo will include the following key
+    subdirectories:</span>
+
+<span class="c11"></span>
+
+<span id="t.11a4ddf8bba3a0a7326318cbd72014efce1ba2dc"></span><span
+id="t.1"></span>
+
+<table border="4" class="c57">
+<colgroup>
```

Comment:
I think we need a bit of explanation of how the project is structured.  Something like.

The example Yavin project consists of the following elements (these can all be source code links):
1. HJSON configuration for 1 or more database.
2. HJSON configuration for 1 or more tables (and their respective measures and dimensions).
3. HJSON configuration for any security roles you want defined.
3. A liquibase script that sets up your database with relevant tables needed for Yavin.
4. Test data that is initialized in the database.
5. Example integration tests that verify the Yavin APIs are working correctly.
6. An application configuration file that configures web service routes, security, and other service level controls.